test/test_turbomole.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Unit testing for Turbomole_class"""
+
+import numpy as np
+import os
+import shutil
+import unittest
+import sys
+from pathlib import Path
+import mudslide
+import yaml
+
+from mudslide import TMModel
+from mudslide.tracer import YAMLTrace
+
+
+@unittest.skipUnless(mudslide.turbomole_model.turbomole_is_installed(),
+        "Turbomole must be installed")
+class TestTMModel(unittest.TestCase):
+    """Test Suite for TMModel class"""
+    
+    def setUp(self):
+        self.turbomole_files = ["control"]
+        for fl in self.turbomole_files:
+            shutil.copy("turbomole_files/"+fl,".")
+
+    def test_get_gs_ex_properties(self):
+        """test for gs_ex_properties function"""
+        model = TMModel(states = [0,1,2,3], turbomole_dir = ".") 
+        positions = model.X 
+
+        mom = [5.583286976987380000, -2.713959745507320000, 0.392059702162967000, 
+                -0.832994241764031000, -0.600752326053757000, -0.384006560250834000, 
+                -1.656414687719690000, 1.062437820195600000, -1.786171104341720000,
+                -2.969087779972610000, 1.161804203506510000, -0.785009852486148000,
+                2.145175145340160000, 0.594918215579156000, 1.075977514428970000,
+                -2.269965412856570000,  0.495551832268249000,   1.487150300486560000]
+
+        traj = mudslide.TrajectorySH(model, positions, mom, 3, tracer = YAMLTrace(name = "TMtrace"), dt = 20, max_time = 21, t0 = 1) 
```

Comment:
So this only does one step? Is that enough to make sure everything is working right? Put another way, do the tests fail if you deliberately introduce a minor bug?